doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -207,16 +205,16 @@ placement groups, execute::
 
 	ceph pg stat
 
-The result should tell you the placement group map version (vNNNNNN), the total
-number of placement groups (x), and how many placement groups are in a
-particular state such as ``active+clean`` (y). ::
+The result should tell you the total
```

Comment:
More a cosmetic nit, but I suggest to rewrap this paragraph.

Reply:
Fixed